public/app/features/search/components/SearchWrapper.tsx

```diff
@@ -28,7 +54,17 @@ export const SearchWrapper: FC = () => {
       appEvents.off(CoreEvents.showDashSearch, openSearch);
       appEvents.off(CoreEvents.hideDashSearch, closeOnItemClick);
     };
```

Comment:
you do not need these events anymore, now anyone can open search by adding the query parameter (see keybindigSrv does this for a lot stuff already top open edit mode or inspect)

Reply:
Some of those events send `payload`, like when you click on dashboard name and the search defaults to current folder, so I think this is necessary until those evens are updated as well.

Reply:
yea, should be just adding a folderId= (or what ever the payload is as url parameter